renderdoc/driver/dxgi/dxgi_common.cpp

```diff
@@ -1391,7 +1391,8 @@ DXGI_FORMAT MakeDXGIFormat(ResourceFormat fmt)
         break;
       case eSpecial_S8: RDCERR("D3D has no stencil-only format"); break;
       case eSpecial_D16S8: RDCERR("D3D has no D16S8 format"); break;
-      default: RDCERR("Unrecognised/unsupported special format %u", fmt.specialFormat); break;
+      default:
+        break;    // Return DXGI_FORMAT_UNKNOWN to convert texture.
```

Comment:
This function has more error cases that need to gracefully return DXGI_FORMAT_UNKNOWN to go down this route properly. For example the RDCERR calls immediately above this default will need to be converted (I would imagine, all depth/stencil formats will decay to D32S8).